CHANGELOG.md

```diff
@@ -6,6 +6,48 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.9.0] - Jan, 5, 2022
+
+### Core
+
+#### Added
+
+- [Koros] Added possibility to rotate Koros
+- [Search Input] Support for placeholder
+
+### React
+
+#### Added
+
+- [Koros] Added possibility to rotate Koros
+- [Search Input] Support for placeholder
+
+#### Changed
+
+- For local development of HDS React, restricted node version to supported versions 12-14
+
+### Design kit
+
+#### Changed
+
+- Updated HDS Sketch libraries to Sketch 81 (81.1)
+
+### Documentation
+
+#### Added
+
+- WP and Drupal notes to the developers’ Getting Started documentation
+- An example of asking the user’s gender in a form to the Form pattern documentation
```

Comment:
in a form to -> in a form **in**

Reply:
The current version is also ok since it is connected to the "added" word.

Reply:
- [ ] Improve grammar